server/ot_rpc.go

```diff
@@ -141,7 +142,11 @@ func runTranscode(n *core.LivepeerNode, orchAddr string, httpc *http.Client, not
 // Orchestrator gRPC
 
 func (h *lphttp) RegisterTranscoder(req *net.RegisterRequest, stream net.Transcoder_RegisterTranscoderServer) error {
-	glog.Info("Got a RegisterTranscoder request for ", req.Secret)
+	from := "unknown"
+	if p, ok := peer.FromContext(stream.Context()); ok {
+		from = p.Addr.String()
+	}
+	glog.Infof("Got a RegisterTranscoder request for %s from %s", req.Secret, from)
```

Comment:
I think printing `req.Secret` might have been a mistake in the original code here.

Should we also tag the log, eg  `glog.Infof("Got a RegisterTranscoder request tc=%s", from)` and use the `tc`  tag (or something similar) throughout when logging transcoder info?

Reply:
Fixed